src/v/config/configuration.cc

```diff
@@ -1476,6 +1476,13 @@ configuration::configuration()
       "set in production.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       false)
+  , disable_cluster_metadata_upload_loop(
+      *this,
+      "disable_cluster_metadata_upload_loop",
```

Comment:
nit: most of our configs are worded the other way around `enable_...`.

Reply:
True, this started out as a `disable_for_tests` kind of property. You're right for features we usually enable